### PR TITLE
KEYCLOAK-11728 New default hostname provider

### PR DESCRIPTION
**Pull request by stianst**

### CODE REVIEW

**Comment by stianst** [comment]
Code: https://github.com/keycloak/keycloak/pull/6388

**Comment by mposolda** [comment]
Did quick look, but delegating to @pskopek for the review.

File: server_admin/topics/threat/admin.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 === Admin Endpoints and Console
 
 The {project_name} administrative REST API and the web console are exposed by default on the same port as non-admin
-usage. If you are exposing {project_name} on the Internet, we recommend not also exposing the admin endpoints on the
+usage. If access to the admin console is not needed externally we recommend not exposing the admin endpoints on the
```
Comment (hmlnarik) [inline server_admin/topics/threat/admin.adoc]:
```suggestion
usage. If access to the admin console is not needed externally, we recommend not exposing the admin endpoints on the
```

File: server_admin/topics/threat/admin.adoc [hunk 2]
```diff
@@ -13,6 +13,9 @@ to `/auth/admin`.
 To achieve this directly in {project_name} there are a few options. This document covers two options, IP restriction
 and separate ports.
 
+Once the admin console is no longer accessible on the frontend URL of Keycloak you need to configure a fixed admin
```
Comment (hmlnarik) [inline server_admin/topics/threat/admin.adoc]:
```suggestion
Once the admin console is no longer accessible on the frontend URL of Keycloak, you need to configure a fixed admin
```

**Review by hmlnarik — DISMISSED** [review]
Looks good, thank you! Two minor points inline which do not block merging from my part

**Pull request merged**